<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>GeometryFactory</author>
 <class>Delaunay_graph_of_ellipses_2</class>
 <widget class="QMainWindow" name="Delaunay_graph_of_ellipses_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGAL 2D Delaunay graph of ellipses</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/cgal/logos/cgal_icon</normaloff>:/cgal/logos/cgal_icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="transformationAnchor">
        <enum>QGraphicsView::NoAnchor</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>File Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Visualization Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRecenter"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionInsert_ellipse"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionI_points"/>
    <addaction name="actionIds"/>
    <addaction name="actionBisectors"/>
    <addaction name="actionDelaunay_edges"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="action_Open"/>
    <addaction name="actionBackground"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAlg">
    <property name="title">
     <string>&amp;Algorithms</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRecenter"/>
    <addaction name="actionCompute_graph"/>
    <addaction name="actionRandom_ellipse"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAlg"/>
   <addaction name="menu_View"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutCGAL">
   <property name="text">
    <string>About &amp;CGAL</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRecenter">
   <property name="icon">
    <iconset>
     <normaloff>:/cgal/Input/zoom-best-fit</normaloff>:/cgal/Input/zoom-best-fit</iconset>
   </property>
   <property name="text">
    <string>Re&amp;center the viewport</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionInnerSkeleton">
   <property name="text">
    <string>Inner Skeleton</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionInsert_ellipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Insert &amp;ellipse</string>
   </property>
  </action>
  <action name="actionI_points">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>i-points</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="actionIds">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ids</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="actionBisectors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Bisectors</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionDelaunay_edges">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Delaunay edges</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>E&amp;xport pdf/ps...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+X</string>
   </property>
  </action>
  <action name="actionCompute_graph">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Compute graph</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionBackground">
   <property name="text">
    <string>Background...</string>
   </property>
  </action>
  <action name="actionRandom_ellipse">
   <property name="text">
    <string>Random ellipse</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
